src/main/java/network/brightspots/rcv/StreamingCVRReader.java

```diff
@@ -338,6 +356,15 @@ public void headerFooter(String s, boolean b, String s1) {
     if (this.unrecognizedCandidateCounts.size() > 0) {
       throw new UnrecognizedCandidatesException(unrecognizedCandidateCounts);
     }
+    // throw if there were data issues
```

Comment:
not trying to nitpick, but I don't think comments like this make the code more readable; your code is already super clear here because you're using descriptive names

Reply:
Yeah, when I wrote the comment there was actually more logic here.